iron-data-table.html

```diff
@@ -399,6 +409,15 @@
         },
 
         /**
+         * If `true`, columns can be reordered by dragging their headers
+         * Column reordering works only if `order` property has been set for each column
```

Comment:
I think we should try to work around this limitation rather than needing explicit `order` values.